Edit meta description, add media (#3929)
Codecademy/docs: content/css/concepts/box-sizing/box-sizing.md

---
Title: 'Box Sizing'

> Description: 'Shorthand property that sets different properties for an elements border in a single declaration.'

Reviewer: ```suggestion
Description: 'Shorthand property that sets different ways to compute width and height for an element in a single declaration.'
```

  box-sizing: content-box;
}
```

## Example 3

> The following image has two `div` containers with a width of 300px, border of 1px and padding of 20px. Inside each container there is a `div` with a width of 100%.

Reviewer: ```suggestion
This output image has two `div` containers with a width of 300px, a border of 1px, and a padding of 20px. Inside each container is a `div` with a width of 100%.
```

> The first container `div` is set to `border-box`. Its total width including the black border, tan padding and blue `div` is 300px.

Reviewer: ```suggestion
The first container `div` is set to `border-box`. Its total width, including the black border, tan padding, and blue `div` is 300px.
```

> The second container `div` is set to `content-box`. The width of the blue content `div` is 300px. The total width of the container `div` with padding and border and content `div` is 342px.

Reviewer: ```suggestion
The second container `div` is set to `content-box`. The width of the blue content `div` is 300px. The total width of the container `div` with padding, border, and content is 342px.
```

> ![Box Sizing Example](https://raw.githubusercontent.com/Codecademy/docs/main/media/box-sizing-example.png 'Box-Sizing Example')

Reviewer: Move this above the explanation.

> ![Box Sizing Example](https://raw.githubusercontent.com/Codecademy/docs/main/media/box-sizing-example.png 'Box-Sizing Example')

Reviewer: ```suggestion
![Illustrates examples 1, and 2 with `box-sizing` set to `border-box`, and `content-box`](https://raw.githubusercontent.com/Codecademy/docs/main/media/box-sizing-example.png)
```
